vms/platformvm/validators/fee/fee_test.go

```diff
@@ -0,0 +1,402 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package fee
+
+import (
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/vms/components/gas"
+
+	safemath "github.com/ava-labs/avalanchego/utils/math"
+)
+
+const (
+	second = 1
+	minute = 60 * second
+	hour   = 60 * minute
+	day    = 24 * hour
+	week   = 7 * day
+	year   = 365 * day
```

Comment:
Not 525,600 minutes?

Reply:
I'm not sure if this comment is implying that `year` should be set to `52 * week` rather than `365 * day` or not haha... But the rest of the P-chain uses this definition of years.

The chain doesn't really need to care about any leap seconds or leap years... We can set our own time cadence... We use this definition of a year for our maximum staking duration (among other things)